src/components/swapv2/SwapSettingsPanel/LiquiditySourcesSetting.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import styled from 'styled-components'
+import { isMobile } from 'react-device-detect'
+import { t, Trans } from '@lingui/macro'
+import { ChevronRight } from 'react-feather'
+import { Flex } from 'rebass'
+
+import QuestionHelper from 'components/QuestionHelper'
+import useAggregatorStats from 'hooks/useAggregatorStats'
+import { useActiveWeb3React } from 'hooks'
+
+type Props = {
+  onClick: () => void
+}
+
+const SettingLabel = styled.span`
+  font-size: ${isMobile ? '14px' : '12px'};
+  color: ${({ theme }) => theme.text};
+  font-weight: 400;
+  line-height: 16px;
+`
+
+const Group = styled.div`
+  display: flex;
+  align-items: center;
+  color: ${({ theme }) => theme.subText};
+  font-size: ${isMobile ? '14px' : '12px'};
+`
+
+const NumberOfSources = styled.span`
+  color: ${({ theme }) => theme.text};
+  font-weight: 400;
+  line-height: 16px;
+`
+
+const LiquiditySourcesSetting: React.FC<Props> = ({ onClick }) => {
+  const { chainId } = useActiveWeb3React()
+  const { data, error } = useAggregatorStats(chainId)
+
+  if (error || !data) {
+    console.error(error)
+    return null
+  }
```

Comment:
This gonna flooding `console.error` if this component keep re-rendering. This will gonna likely to be happened because our codebase is not optimize well in times of re-rendering.

Please remove `console.error`, left only `return null`. If you still want to log it, move it into `useEffect`

Also, please add some meaningful message too. Dont log only `error`

Reply:
Yeah, I removed it.